### ISSUE TITLE
Bylaws document feedback

### ISSUE THREAD

**Issue by sepu85** [pythonpete32/aragoncoop_governance#11]
this issue intends to feed back the content exposed in the [proposing bylaw's doc](https://github.com/pythonpete32/aragoncoop_governance/blob/master/bylaws.md)

> Section 2    Purpose
> The organization is incorporated solely for charitable purposes which is:
> 
> To support grassroots community initiatives in the broader Aragon ecosystem. This will include but not limited to general documentation and guides, demos, workshops, and meetups, with the purpose of helping people understand what Aragon is and how they can build with it. Its main focus is on engaging with people, talking to them, and distilling their questions and comments into actionable insights for the broader Aragon community.

I feel there's a general overlooking (this is referred to the whole community, not to the wording proposed above) about the importance of the **purpose**, this is what would unite us as a single being/entity/organization and we ALL should be included in it's definition.

The Coop was born as a response to the lack of engagement of ANT holders during ANVs, which demonstrate that there is a huge room for improvement as regards Aragon governance system, therefore I'm pretty convinced Aragon Coop purpose should be oriented to test by itself new and different governance mechanism and propose meaningful changes to Aragon's gov mechanism.

Despite ANVs, I think there might be some technocracy in the way things are decided in the current Aragon's ecosystem, I believe the Aragon Coop could be purposed to bring balance to that equation by adding more humanity (which means considering people's interests, talents, sentiments, as well as their vulnerabilities as fears, egos, ignorance) in the way things are decided.

The above is of course my biased view about what should be the purpose, we still need a way to engage the whole community in a purpose definition dynamic where we break down the individual biased view about the Coop purpose, aggregate all that information and jointly define a single robust purpose that unites us all in walking the same path to build the best possible frameworks for building the decentralized organizations of the future. 

About Membership, please see my comments issued [here](https://github.com/aragoncoop/temp/issues/2#issuecomment-496951636). 

Also, I think the the governance shall be described in these bylaws again, but this should be again, defined by the whole community, I made a proposal asking each member to follow instructions on [this excel doc](https://github.com/aragoncoop/temp/issues/2#issuecomment-496951636), I would insist you in giving a chance as well as thinking in a way to engage with the community and build this Coop fundamentals all together.

**Comment by pythonpete32** [comment]
Thanks for the feedback  @ its really important to get as many views as possible before submitting this for a general vote 🙌

> I feel there's a general overlooking (this is referred to the whole community, not to the wording proposed above)

General comments are best left for the keybase chat. The purpose of these discussions should be Laser focused on the documents at hand. 

> the importance of the purpose , this is what would unite us as a single being/entity/organization and we ALL should be included in it's definition.
Sure totally agree, but who exactly is being excluded. What would be helpful is if you suggest some edits to this section of the document and see if we can get the wording to everyone's liking. Either in the comments here or in a PR


> DespiteANVs, I think there might be some technocracy in the way things are decided in the current Aragon's ecosystem, I believe the Aragon Coop  to bring balance to that equation by adding more humanity (which means considering people's interests, talents, sentiments, as well as their vulnerabilities as fears, egos, ignorance) in the way things are decided.


I agree, Aragon is being built by technologist, I also see the coop as being a counterbalance to this. However again we should keep the discussion laser-focused on the document in question. Of we can try to keep the criticisms actionable we can move forward in the same direction


>, therefore, I'm pretty convinced Aragon Coop purpose should be oriented to test by itself new and different governance mechanism and propose meaningful changes to Aragon's gov mechanism.

I also agree, testing governance systems is important, this could be added to the purpose along with the rest. You can submit a PR yourself or ill add it when I collect more feedback


> The above is of course my biased view about what should be the purpose, we still need a way to engage the whole community in a purpose definition dynamic where we break down the individual biased view about the Coop purpose, aggregate all that information and jointly define a single robust purpose that unites us all in walking the same path to build the best possible frameworks for building the decentralized organizations of the future.

There is no such thing as an unbiased view 😄 every view is subjective to the individual but that does not diminish its value. 

What we are doing right now is engaging the whole of Our coop community. This document was produced by a small subset of the coop, everyone has the opportunity to raise there objections and suggestions to specific parts of it just like your doing now.

It's worth remembering, this is just a proposal. It is not yet official coop policy. The broader the consensus we reach this stage, the easier it will pass later. But equally, if this is not something the coop at large wants, it will not pass a vote. It is equally beholden on members who want to go in another direction to help shape this document or produce something else for review. That's how we move forward.


> Also, I think the governance shall be described in these bylaws again, but this should be again, defined by the whole community, 

Again, this is a proposal of a subset of the coop. Anyone disagrees with the definitions, either bring a concrete amendment in the form of an issue or PR. the other option is to just produce a counterproposal. I am not attached to anything other than the need for action.

**Comment by burrrata** [comment]
Idea: maybe the "Objective" should be it's own document? As is the Bylaws mostly talk about stuff like when we meet and how things get done. The "Objective" though is something a lot of people feel strongly about and want to weigh in on. Since the Bylaws already references several external documents, maybe there could be an external "Objective" document as well? This would allow conversations around the bylaws to stay concrete while more philosophical discussions could be move to the temp repo Issue 3: https://github.com/aragoncoop/temp/issues/3

**Comment by pythonpete32** [comment]
@burrrata I agree, decoupling it from the bylaws makes sense, the bylaws are primarily about rules. The objectives of the coop are rightly what people are passionate about

@sepu85 it would be great if you could take the lead on this

**Comment by sepu85** [comment]
@pythonpete32 I am really looking forward to lead this, I'm still waiting for you guys to participate in my proposal regarding [this excell doc](https://docs.google.com/spreadsheets/d/1wUVZ6pSeaQtI2_JVn0Bsli47WA4ba939Tyl9FXXxmrE/edit?usp=sharing) before I go to the rest of the community and seek for more engagement.

**Comment by burrrata** [comment]
The official governance process of the Aragon Cooperative is the temp repo, not an excel spreadsheet. If you don't want to participate in the governance process the group has agreed to, then you should probably leave. If you do want to participate, then please either join in discussions in the GitHub Issues or submit a Meta Proposal to change the current governance process.
